mention-bot.js

```diff
@@ -285,12 +303,17 @@ function guessOwners(
       return owner !== creator;
     })
```

Comment:
In the future, I imagine we will add more config options for user eligibility (for example, required organizations), so I think it makes sense having this be its own function.